### ISSUE TITLE
support for _events_

### ISSUE THREAD

**Issue by cdaringe** [jtanx/libclipboard#11]
hello,

is there consideration for adding change events to this lib?

**Comment by jtanx** [comment]
No, it hasn't been considered, but I guess it wouldn't be too bad to add. No guarantees on when it'd be added though; in the mean time, patches are welcome :)
